opensrp-chw/src/at/assets/json.form/adolescent_hv_medication_in_use_sw.json

```diff
@@ -0,0 +1,587 @@
+{
+  "count": "1",
+  "encounter_type": "Tembeleo la Kijana Balehe - Utumiaji wa madawa",
```

Comment:
I dont remember how this works, do we need to translated the encounter type? I think from the data's perspective, we are better off having the same names for encounter types

Reply:
🤔 I remember that, I think we can keep it the same name. Let me change that and update